db/schema.rb

```diff
@@ -247,10 +248,10 @@
     t.string "scopes", default: "", null: false
     t.string "lead_application_source", default: "", null: false
     t.boolean "confidential", default: true, null: false
-    t.boolean "can_access_private_user_data", default: false, null: false
-    t.boolean "can_find_or_create_accounts", default: false, null: false
-    t.boolean "can_message_users", default: false, null: false
-    t.boolean "can_skip_oauth_screen", default: false, null: false
+    t.boolean "can_access_private_user_data", default: false
+    t.boolean "can_find_or_create_accounts", default: false
+    t.boolean "can_message_users", default: false
+    t.boolean "can_skip_oauth_screen", default: false
```

Comment:
I don't think these should've changed. Not in this PR at least. 

🤔 ... could it be my fault from a previous PR? I would suggest recreating your local DB.

Reply:
Not sure where this came from. It was not part of the conflict with master, but is fixed now.